spec/setup_acceptance_node.pp

```diff
@@ -4,3 +4,7 @@
     ensure => present,
   }
 }
+
+user { 'fwernli':
```

Comment:
Why ~is this user~ am I needed here ?

Reply:
I wanted to keep the tests unchanged and just rename them. Feel free to add commits to the branch, just add comments each time to do so because I trend to force push a lot ATM because many branches are pending :-)

Reply:
These changes looks good to me :-)